### PR TITLE
Review documentation

### PR DESCRIPTION
**Pull request by xamanu**

### CODE REVIEW

File: docs/user/requirements.md [hunk 1]
```diff
@@ -1,13 +1,25 @@
 ---
 Title: Requirements
-Description: I can use my 7 years old computer to run the K-Box?
+Description: Hardware and Software prerequisites
```
Comment (avvertix) [inline docs/user/requirements.md]: You really want this boring default string to feed the search engine browsing our documentation?

File: readme.md [hunk 2]
```diff
@@ -2,34 +2,48 @@
...
+K-Box can be installed on most operating systems. The setup is heavily based on [Docker](https://www.docker.com/).
 
-The K-Box code is covered by unit tests.
+* See the [system requirements](./docs/user/requirements.md).
+* Follow these simple steps to [install K-Box](./docs/user/installation.md).
 
-For more information see [Executing Unit Tests](./docs/developer/testing.md).
+### Components
 
-## Contributing
```
[the remark this answers is not in the document]
Comment (xamanu) [reply readme.md]: Into the first sentence and the Open Source part.

File: docs/user/requirements.md (hunk 1 above)
Comment (xamanu) [reply docs/user/requirements.md]: :) Probably yes.

File: docs/user/configuration.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
+# Configuration
```
[the remark this answers is not in the document]
Comment (xamanu) [reply docs/user/configuration.md]: We can always create a linked `advances_configuration.md` or extend this one here.

**Comment by avvertix** [comment]
@xamanu I realized that there is a problem in the simple installation on the readme. It tells to create only one folder, but considering that the docker-compose has configured volumes, the startup will fail immediately

**Pull request merged**